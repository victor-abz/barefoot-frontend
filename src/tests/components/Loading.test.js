/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loading from '../../components/common/loading';

describe('Test On Loading Component', () => {
	const wrapper = shallow(<Loading />);
	describe('render()', () => {
		test('Should Render Loading Component', () => {
			expect(toJson(wrapper)).toMatchSnapshot();
		});
	});
});
