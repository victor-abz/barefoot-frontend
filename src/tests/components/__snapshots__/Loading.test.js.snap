// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test On Loading Component render() Should Render Loading Component 1`] = `
<div
  className="lds-ellipsis"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
